packages/server-wallet/src/engine/__test__/integration/push-message.test.ts

```diff
@@ -25,7 +27,7 @@ import {MultiThreadedEngine, Engine} from '../..';
 const dropNonVariables = (s: SignedState): any =>
   _.pick(s, 'appData', 'outcome', 'isFinal', 'turnNum', 'stateHash', 'signatures');
 
-jest.setTimeout(20_000);
```

Comment:
Was this timeout increased while playing around with test? Or do we need the longer timeout?

Reply:
Ah, good catch. This was changed when I was originally investigating the issue and I forgot to modify it back.

Reply:
b9cac13